### ISSUE TITLE
Improve search results by displaying results from api, blog posts and guides separately

### ISSUE THREAD

**Issue by acorncom** [emberjs/guides#1093]
Per a suggestion by @szimek at https://github.com/emberjs/website/issues/2213, it'd be lovely to have search results split results by type:

> I'm not sure if it's possible with Swiftype, but it would be really great.

> I was looking for a way to scroll page to the top on route transition, but only if the previous and the next route are not from the same "section". I found info about Route and its willTransition event and that Transition object is passed to the handler. However, finding any info about Transition object itself in docs or using search is impossible. If I type "transition" into the search box, I get links to blog posts and guides, but I can't see anything related to API docs. Maybe this object is not documented, but if the search results were separated, it would be much easier for me to find out.

**Comment by michaelrkn** [comment]
I'm taking "help wanted off" since this isn't something the general community can help with. We need somebody who has access to our Swiftype account to dig around to figure this out.

**Comment by acorncom** [comment]
Good point.  For the record, who has access to our Swiftype account? ;-)

**Comment by michaelrkn** [comment]
Me and locks and you shortly :)

**Comment by acorncom** [comment]
Once we've merged our Algolia work ( #1184 ), we'll work on adding in search of the blog posts as well.
